minkyu/express_init.js

```diff
@@ -0,0 +1,41 @@
+const http = require('http');
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const dotenv = require('dotenv');
+dotenv.config();
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env. TYPEORM_CONNECTION,
+    host: process.env. TYPEORM_HOST,
+    port: process.env. TYPEORM_PORT,
+    username: process.env. TYPEORM_USERNAME,
+    password: process.env. TYPEORM_PASSWORD,
+    database: process.env. TYPEORM_DATABASE,
+})
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!")
+    });
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(''));
+
+app.get("/ping", (req,res) =>{
+    res.json({message : "pong"});
+});
+
+const server = http.createServer(app);
+const PORT = process.env.PORT;
+
+const start = async () => {
+    server.listen(PORT, () => console.log(`server is listenning on ${PORT}`));
```

Comment:
express 객체가 `app` 이라는 변수에 할당되어 있습니다.
미들웨어 역시 `app.use(...)`의 형태로 app의 미들웨어로 지정되고 있기 때문에 app 자체를 Listen()하면 될 것 같습니다!

```suggestion
    app.listen(PORT, () => console.log(`server is listenning on ${PORT}`));
```